---
layout: page
title: Programs
permalink: /Programs/
---


### About

The lab does not maintain all software on a single github account, programs developed are spread across the developers personal account.

This page highlights some of the programs that have either been developed by folks in the lab or are currently in development, feel free to check them out and follow lab folks on their githubs for updates.

[Semblans](https://github.com/gladshire/Semblans/): Led by Miles Woodcock-Girard, this C++ program provides an automated way to assemble transcriptomes. The paper has now been published in bioinformatics and can be found [here](https://academic.oup.com/bioinformatics/article/41/1/btaf003/7950665).

[CAnDI](https://github.com/HollyMaeRobertson/CAnDI): Led by Holly Robertson and developed in collaboration with the Moyroud group, this Python program dissects large gene families allowing you to perform conflict analyses without having to first do orthology detection. You can also search large gene families for patterns of conflict that may help explain trait evolution in the tree of life. The paper can be found [here](https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syaf028/8126907?login=false).

[TreeHouse](https://github.com/karolisr/treehouse): Led by Karolis Ramanauskas and currently in development, however, it does have working releases. Treehouse simplifies tree visualization and lets you easily view and navigate extremely large trees. New features are being added regularly, so please open an issue and let Karolis know if there's anything you'd like in a tree viewer.

[TransRate2](https://github.com/ericbretz/transrate2): Led by Eric Bretz and in collaboration with Dr. Steve Kelly, this version of TransRate incorporates several new features and updates to the mapping procedure. This is currently in development.

[TreeForge](https://github.com/ericbretz/TreeForge): Led by Eric Bretz, this program takes in transcriptomes to perform a full phylotranscriptomic analysis following the [Yang and Smith, 2014](https://bitbucket.org/yanglab/phylogenomic_dataset_construction/src/master/) procedure. The program incorporates a [phylogenetically informed hierarchichal clustering algorithm](https://bsapubs.onlinelibrary.wiley.com/doi/10.1002/ajb2.1069) to help cut down on computational time. This is currently in development.

[MissMap](https://github.com/zawmbi/MissMap): Led by Linda Mansour, this program uses Large Language Models to help navigate some of the cluter on NCBI. It is currently in development.
