---
layout: page
title: Research
permalink: /Research/
---

## Phylogenetics and phylogenomics

The lab has several projects focused on phylogenetics and phylogenomics. This includes exploring phylogenomic approaches such as in this paper, as led by Alexa and Eric, along with phylogenomic analyses such as those conducted by Holly and Tomi. One specific area is examining gene tree conflict and its role in our studies, both from analyzing the support for species relationships and gaining a deeper understanding of the biological implications of conflict. Recently, Holly found bias in the number of genes supporting morphological convergence compared to an equally evolutionary relationship; more info [here](https://www.biorxiv.org/content/10.1101/2023.11.18.567661v1.abstract). Holly also developed a program to help with analyses, available [here]( https://github.com/HollyMaeRobertson/CAnDI).

## Obtaining and analyzing transcriptomic data

The lab has been focused on obtaining transcriptomes from tissue that hasn’t been preserved for this purpose. Natalia, a graduate student in Robie’s lab, conducted a thesis chapter in the lab where she [extracted RNA from silica-preserved tissues](https://www.biorxiv.org/content/10.1101/2023.06.16.545126v1.abstract). We hope to refine protocols and methods to obtain transcriptomes from as many tissues as possible so herbaria collections can help fill the sampling gap. Alexa’s putting together a review on the bias that current transcriptome sampling produces and what we may do to alleviate these biases. We’ve also examined how transcriptome data may be applied to population genetics questions.

<br>

![NataliasPaper](https://walkerlab-uic.github.io/pictures/NataliaPaperFigure.png)

This image shows the samples Natalia collected across the Dominican Republic for her paper. Pie charts are labelled with sample numbers and show the proportion of each layer inferred to have contributed to the ancestry of each sample based on the conStruct analysis. (a) Darker areas correspond to higher elevations, with major mountain ranges labeled. (b) The Fast Estimation of Effective Migration Surfaces (FEEMS) analysis. Warmer colors correspond to areas inferred to have less gene flow.


<br>

## Processing next-generation sequencing data

The lab has several projects that involve developing novel programs to process and analyze genomic data. This includes Miles' work on assembly, which can be found [here]( https://github.com/gladshire/Semblans), and Eric's work on analyzing dataset quality.

## Urban Ecology and Evolution

This is early work being led by Shawn. Specifically, it examines how a major urban environment may apply selective pressures on the species that grow in it.

